src/Umbraco.Infrastructure/Runtime/CoreInitialComposer.cs

```diff
@@ -204,13 +203,6 @@ public override void Compose(Composition composition)
             // Config manipulator
             composition.RegisterUnique<IConfigManipulator, JsonConfigManipulator>();
 
-
-            // register the http context and umbraco context accessors
-            // we *should* use the HttpContextUmbracoContextAccessor, however there are cases when
-            // we have no http context, eg when booting Umbraco or in background threads, so instead
-            // let's use an hybrid accessor that can fall back to a ThreadStatic context.
-            composition.RegisterUnique<IUmbracoContextAccessor, HybridUmbracoContextAccessor>();
-
```

Comment:
Not sure I understand why this is removed?

Reply:
It was registered twice.

Reply:
Then it makes perfect sense 💪  😄